litmus-portal/frontend/src/views/Settings/ImageRegistry/index.tsx

```diff
@@ -346,9 +368,43 @@ const ImageRegistry = () => {
                     {registryType === 'Private' && (
                       <>
                         <hr />
-                        <Typography className={classes.additionalDetails}>
-                          {t('settings.imageRegistry.additionalInfo')}
-                        </Typography>
+                        <div style={{ display: 'flex' }}>
+                          <Typography className={classes.additionalDetails}>
+                            {t('settings.imageRegistry.additionalInfo')}
+                          </Typography>
+                          <IconButton
+                            className={classes.iconBtn}
+                            onClick={handleClick}
+                            aria-label="info"
+                          >
+                            <InfoIcon />
+                          </IconButton>
+                          <Popover
+                            id={id}
+                            open={open}
+                            anchorEl={anchorEl}
+                            onClose={handleClose}
+                            anchorOrigin={{
+                              vertical: 'bottom',
+                              horizontal: 'center',
+                            }}
+                            transformOrigin={{
+                              vertical: 'top',
+                              horizontal: 'center',
+                            }}
+                          >
+                            <Typography className={classes.infoText}>
+                              {t('settings.imageRegistry.provideInfo')}{' '}
+                              <a
+                                href="https://kubernetes.io/docs/concepts/configuration/secret/"
+                                target="_"
+                              >
+                                {t('settings.imageRegistry.visit')}
+                              </a>
+                              .
```

Comment:
is this needed?

Reply:
The document url?

Reply:
The '.' @amityt ?

Reply:
Ohh, that full stop wont cause any issue @ishangupta-ds

Reply:
I think this is a full-stop at the end of the tooltip

Reply:
Attaching screenshot for reference
![Screenshot from 2021-05-14 16-10-17](https://user-images.githubusercontent.com/40661238/118265297-932af880-b4d6-11eb-967c-5eeba6301682.png)

Reply:
Could have been in translation itself @amityt